.github/workflows/ci.yml

```diff
@@ -80,14 +80,26 @@ jobs:
     runs-on: ubuntu-22.04
     needs: publish-docker
     steps:
```

Comment:
@kofrdx you'll need to add permissions block here as you did in postman.yml file